### PR TITLE
fix for uploading iPhone X screenshots

### PR DESCRIPTION
**Pull request by cherpake**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

Fix to upload screenshots for iPhone X

### Description

added some values here and there

### CODE REVIEW

**Comment by cherpake** [comment]
I signed it!

File: spaceship/lib/spaceship/tunes/device_type.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Spaceship
   module Tunes
     class DeviceType
-      @types = ['iphone4', 'iphone35', 'iphone6', 'iphone6Plus', 'ipad', 'ipadPro', 'watch', 'appleTV', 'desktop']
+        @types = ['iphone4', 'iphone35', 'iphone6', 'iphone6Plus', 'iphone58', 'ipad', 'ipadPro', 'watch', 'appleTV', 'desktop']
```
Comment (KrauseFx) [inline spaceship/lib/spaceship/tunes/device_type.rb]: Was the spacing change here intentional?

**Comment by KrauseFx** [comment]
https://github.com/fastlane/fastlane/pull/10731

File: spaceship/lib/spaceship/tunes/device_type.rb (hunk 1 above)
Comment (cherpake) [reply spaceship/lib/spaceship/tunes/device_type.rb]: Nope, sorry

**Comment by nafu** [comment]
@cherpake Thank you for your contribution!

I’m sorry I missed your PR and I already created #10731.
And thank you for reviewing #10731 👍

**Comment by cherpake** [comment]
Hey no problem :)

**Comment by ohayon** [comment]
Hey @cherpake - we just made a release that supports iPhone X uploads. Thank you so much for this contribution though! We would love to see more from you! 🚀

**Pull request closed without merging**